reflectivity_ui/interfaces/event_handlers/configuration_handler.py

```diff
@@ -1,20 +1,37 @@
+# local imports
+from reflectivity_ui.interfaces.configuration import Configuration
+
+# third party imports
 from PyQt5.QtCore import Qt
-from PyQt5.QtWidgets import QSpinBox, QCheckBox, QDoubleSpinBox
+from PyQt5.QtWidgets import QSpinBox, QCheckBox, QDoubleSpinBox, QWidget
 
-from reflectivity_ui.interfaces.configuration import Configuration
+# standard imports
+from dataclasses import dataclass
+
+
+@dataclass
+class ConfigWidget:
```

Comment:
instances of this class are only created inside function `connect_config_events()` so in order to limit the scope, could you include this within that function?
Also include a docstring, please :slightly_smiling_face: